### DISCUSSION TITLE
Choose the version on shell.duckdb.org

### DISCUSSION THREAD

**Discussion by fsemecurbe** [duckdb/duckdb#14382] [Ideas]
Duck is a highly relevant tool for data scientists. I have to present duckdb from shell.duckdb.org to my colleagues. With each new version, the online version no longer allows you to launch the space extension. Can I choose the version of duckdb in the online version?

**Comment by carlopi** [comment]
This is a transient problem connected to the release process, that we have to smoothen up. I will add spatial as hard dependency on bumping duckdb-wasm, this seems more logical.

Request does make sense. Thanks!
